inference-engine/tests/ie_test_utils/common_test_utils/data_utils.hpp

```diff
@@ -139,6 +140,49 @@ fill_data_roi(InferenceEngine::Blob::Ptr &blob, const uint32_t range, const int
     }
 }
 
+template<InferenceEngine::Precision::ePrecision PRC>
+inline void
+fill_data_roi(ov::runtime::Tensor& tensor, const uint32_t range, const int height, const int width, const float omega,
```

Comment:
I guess we need to add some description for this function.

Reply:
Discussed offline. The description will be added as part of the next activities